core/src/scene/sceneLoader.cpp

```diff
@@ -1160,14 +1158,15 @@ Filter SceneLoader::generateNoneFilter(Node _filter, Scene& scene) {
 }
 
 void SceneLoader::parseStyleParams(Node params, Scene& scene, const std::string& prefix,
-                                   std::vector<StyleParam>& out) {
+                                   std::vector<StyleParam>& out, bool unified) {
 
     for (const auto& prop : params) {
+        unified |= prefix == "text";
         std::string key;
```

Comment:
Yeah ... maybe will use the default params for text builder.
Can you also try how this behaves on JS?